3rdparty/openenclave/ert.patch

```diff
@@ -689,11 +765,18 @@ index cf814d36a..5de0c930d 100644
 +        OE_CHECK(oeimage_payload.add_pages(
 +            &oeimage_payload, context, enclave, &vaddr));
 +        context->payload_offset = image_size;
++
++        OE_CHECK(_add_readonly_pages(
++            context,
++            enclave->addr,
++            &vaddr,
++            (uint8_t*)oeimage_payload.elf.elf.data + payload_data_offset,
```

Comment:
Maybe define this in an extra variable, instead of adding it in the call for better code readability?